numpy/linalg/tests/test_linalg.py

```diff
@@ -682,6 +682,12 @@ def test_empty_identity(self):
         assert_equal(vh.shape, (4, 4))
         assert_equal(vh, np.eye(4))
 
+    def test_svdvals(self):
+        x = np.array([[1, 0.5], [0.5, 1]])
+        s_from_svd = linalg.svd(x, compute_uv=False, hermitian=False)
```

Comment:
The implementation doesn't explicitly set `hermitian=False`, perhaps it should if the test does as well?

Reply:
That's true - `svdvals` now calls `svd` with `hermitian=False`.